5-membership.md

```diff
@@ -20,11 +20,11 @@ Why this process?
 
 *The members create and maintain the hacker environment.*
 
-When a conflict/problem can not be resolved between individuals/via do-ocracy or when it impacts the group, a group decision is required. Any member can request that a decision is made by the Group Of Members instead of by do-ocracy/individual members.
+When a conflict/problem can not be resolved between individuals, via do-ocracy or when it impacts the group, a group decision is required. Any member can request that a decision is made by the group of members instead of by do-ocracy or individual members.
```

Comment:
"group of members" language isn't used anymore in the document. I just call it "meetings" now.

Reply:
so "is made on a meeting instead of by do-ocracy"